examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+    gtab = dpg.gradient_table(hardi_fbval, hardi_fbvec)
+
+    warped_hardi, mapping = reg.syn_register_dwi(hardi_fdata, gtab)
+    reg.write_mapping(mapping, op.join(working_dir, 'mapping.nii.gz'))
+else:
+    mapping = reg.read_mapping(op.join(working_dir, 'mapping.nii.gz'),
+                               img,
+                               MNI_T2_img)
+
+mapping_img = nib.load(op.join(working_dir, 'mapping.nii.gz'))
+mapping_img_data = mapping_img.get_fdata()
+
+# Working with diffeomorphic map data
+mapping_img_data = mapping_img_data[..., 0, 0]
+show_anatomical_slices(mapping_img_data, 'Registration Displacement Mapping')
+
+# plot the transformational map of MNI T2 onto subject space
+warped_MNI_T2_data = mapping.transform_inverse(MNI_T2_img.get_fdata())
+warped_MNI_T2_img = nib.Nifti1Image(warped_MNI_T2_data.astype(float),
+                                    img.affine)
+
+nib.save(warped_MNI_T2_img, op.join(working_dir, 'warped_MNI_T2.nii.gz'))
+
+show_anatomical_slices(warped_MNI_T2_img.get_fdata(), 'Warped MNI T2')
+
+##########################################################################
+# Create novel bundle specification:
+# ----------------------------------
+# The bundles specify meta-data for the segmentation.
+#
+# The following keys are required for a `bundle` entry:
+#
+# - `ROIs`
+#
+#   lists the ROI templates.
+#
+# .. note::
+#   Order of the `ROIs` matters and may result in different tract profiles
+#
+# - `rules`
+#
+#   label each ROI as inclusionary `True` or exclusionary `False`.
+#
+# - `cross_midline`
+#
+#   whether or not the streamline crosses the midline
+#
+# .. note::
+#
+#   It is also possible to utilize probablity maps to further refine ROIs. If
+#   `prob_map` key is not specified the probablities will all be ones and same
+#   shape as the ROI.
+#
+# .. note::
+#
+#   If using multiple bundles it is recommended to add an optional identifier
+#   key such as `uid`. This key is not used by segmentation, but can be helpful
+#   for debugging or quality control reports.
+#
+# To identify candidate streamlines for the anterior forceps we will use three
+# waypoint ROI templates:
+#
+# 1. Left anterior frontal,
+#
+# 2. Right anterior frontal,
+#
+# 3. and midsaggital.
+#
+# The templates are first resampled into the MNI space, before
+# they are brought into the subject's individual native space.
+
+
+print("Fetching callosum ROI templates...")
+
+callosal_templates = afd.read_callosum_templates(resample_to=MNI_T2_img)
+
+show_anatomical_slices(callosal_templates["L_AntFrontal"].get_fdata(),
+                       'Left anterior frontal ROI')
+show_anatomical_slices(callosal_templates["R_AntFrontal"].get_fdata(),
+                       'Right anterior frontal ROI')
+show_anatomical_slices(callosal_templates["Callosum_midsag"].get_fdata(),
+                       'Midsagittal ROI')
+
+print("Creating callosal bundle specification...")
+
+# bundle dict
+bundles = {}
+
+# anterior frontal ROIs
```

Comment:
Incorporate bundle specification definitions for all callosum templates into api.

Reply:
Added as enhancement request #494